### PR TITLE
Bundle pro to final bundles

### PR DESCRIPTION
**Pull request by adrinr**
## Description
When running locally for OSS users, we removed the references of `pro` and `backend-core` to allow everything to run fine. Without this, we are having issues between the bundled `backend-core` and the one used by the `pro` npm package. This worked fine while running locally, but it did not allow running docker files locally.

This PR ensures that the pro package is always bundled, both when running as OSS or with the submodule loaded.

Checks:

- [x] Run `yarn dev` locally with the submodule loaded
- [x] Run `yarn dev:docker` locally with the submodule loaded
- [x] Build and run `single image` (via ./scripts/build-single-image.sh) with the submodule loaded
- [x] Run `yarn dev` locally without the submodule loaded
- [x] Run `yarn dev:docker` locally without the submodule loaded
- [x] Build and run `single image` (via ./scripts/build-single-image.sh) without the submodule loaded
- [x] Run `OSS pipeline`: https://github.com/Budibase/budibase/actions/runs/10150562387
- [x] Run as a `feature branch`

OSS pipeline

## Launchcontrol
Allow OSS to run the docker images locally without pro issues

## Feature branch env
[Feature Branch Link](http://fb-chore-fix-oss-docker-issues.fb.qa.budibase.net)

### CODE REVIEW

File: scripts/bootstrap.sh
Comment (adrinr) [inline scripts/bootstrap.sh:1]: This is a legacy script, not used in ages

File: scripts/devDocker.sh [hunk 1]
```diff
@@ -1,11 +1,5 @@
 #!/bin/bash
 
-# Check if the pro submodule is loaded
```
Comment (adrinr) [inline scripts/devDocker.sh:3]: Now we can run this! :)

File: scripts/build.js [hunk 2]
```diff
@@ -75,7 +61,7 @@ async function runBuild(entry, outfile) {
       svelteCompilePlugin,
       TsconfigPathsPlugin({ tsconfig: tsconfigPathPluginContent }),
       nodeExternalsPlugin({
-        allowList: ["@budibase/frontend-core", "svelte"],
+        allowList: ["@budibase/frontend-core", "@budibase/pro", "svelte"],
```
Comment (adrinr) [inline scripts/build.js:64]: Pro will be now bundled, from the source code with the submodule loaded or from node_modules for OSS users

**Review by samwho — APPROVED** [review]

**Review by shogunpurple — APPROVED** [review]
Awesome work on this Adri, been a hassle for a while!

**Review by mike12345567 — APPROVED** [review]
Very nice - very nice that we can fully mimic our method with this!

**Comment by mike12345567** [comment]
Addresses: https://github.com/Budibase/budibase/issues/14260

**Pull request merged**